src/els_db.erl

```diff
@@ -12,7 +12,8 @@
         , stop/0
         , tables/0
         , transaction/1
-        , write/1
+        , write/1,
```

Comment:
The project currently uses a comma-first indentation. Things may change in the future, for now please follow the guidelines at https://github.com/erlang-ls/erlang_ls/blob/master/.github/CONTRIBUTING.md

Reply:
Also, why exporting it?

Reply:
Ah, sorry, I did this just for testing, that's also why I didn't adhere to the guidelines. Will fix.